src/components/TranslationPanel.js

```diff
@@ -11,11 +11,17 @@ const i18n = new(require('../translations/i18n'));
 const db = require(`${__dirname}/../util/data-provider`).targetDb();
 import Statistic  from '../components/Statistic';
 import { FormattedMessage } from 'react-intl';
+import { CommunicationSpeakerPhone } from 'material-ui/svg-icons';
```

Comment:
What is this used for?

Reply:
Sorry by mistaken this has been pushed. This happen while i was testing the code it got imported automatically when i use some elements down the page. So now on-wards before PR i will recheck all the files perfectly..